CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to saskatoon-ng
+
+Contributions are more than welcome!
+
+## Setup
+
+Please follow instructions in `INSTALL.md` to set up your local development environment.
+
+
+## Branches
+
+Unless instructed otherwise by maintainers all pull requests should target the `develop` branch.
+
+If your branch directly addresses an opened `Issue`, please name your branch using the following scheme: `Fix-<Issue#>-A_short_description`
+
+Maintainers may create specific `Feature-<Issue#>-A_short_description` branches for larger issues so contributors can merge their patches into these branches before they get integrated into `develop`.
+
+
+## Commits
+
+Before finalizing your pull request, squash your commits into a minimal number of relevant commits.
+
+> TODO: include some commit messages guidelines (or link to a reference)
```

Comment:
@2ynn 
How about using imperative form in commit messages? Like `Fix some-bug` instead of `Fixed`. As that's how git makes commit messages when using git merge, same for github iirc.

Maybe if it's on this form: `Add/Refactor/Test/Docs/Remove commit-message`.
You and tristan write commits in different ways though so it depends on your convention.

Maybe also **not** use `[#issue-number] commit-message`, because we can reference issues in PRs.

You can check [this freecodecamp article](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) too if would like to. There are some useful guidelines there too.

Tell me what you think of my suggestion and I'll add some edits.

Reply:
oh yes my commit messages are terrible, i definitely need guidelines ;) yes imperative form is good! 
ok for not including issue number in the PR title and referencing it in the comment section. To me having the issue number in commits though could help when looking back at the git log history. But maybe it doesn't make sense.

Reply:
Also how about adding [WIP] before a PR's name to let you know it's not finished yet and tell the contributors to mention you, share screenshots (I could edit the PR template to include maybe screenshots, links to pages, bullet points to what they did exactly, etc...)

Reply:
can we simply use the `Draft` functionality for marking PRs as not ready?

Reply:
Yes using draft sounds better.